tests/system/action/personal_note/test_create.py

```diff
@@ -1,14 +1,20 @@
+from typing import Any, Dict
+
 from tests.system.action.base import BaseActionTestCase
 
 
 class PersonalNoteCreateActionTest(BaseActionTestCase):
+    def setUp(self) -> None:
+        super().setUp()
+        self.test_model: Dict[str, Dict[str, Any]] = {
+            "meeting/110": {"name": "name_meeting_110"},
+            "motion/23": {"meeting_id": 110},
+            "user/1": {"meeting_ids": [110]},
+        }
+
     def test_create(self) -> None:
-        self.set_models(
-            {
-                "meeting/110": {"name": "name_meeting_110"},
-                "motion/23": {"meeting_id": 110},
-            }
-        )
+        # checks permissions too.
```

Comment:
What do you mean with this comment?

Reply:
Never mind, it was a reminder, that this test also checks permissions.